.gitignore

```diff
@@ -1,5 +1,2 @@
-test-db/
-test-db-*/
 coverage
-node_modules/
-*.log
```

Comment:
I think best practice is to put a `.gitignore` file in your users home directory (`~/.gitignore`). Put `.DS_Store` in there, so the separate projects don't need it. 

What is `*.log` for? We don't have log files in this module, do we?